src/edfi-performance-test/edfi_performance_test/api/client/v5/student.py

```diff
@@ -12,23 +12,44 @@
 class StudentContactAssociationClient(EdFiAPIClient):
     endpoint = "studentContactAssociations"
 
-    dependencies: Dict = {
-        "edfi_performance_test.api.client.student.StudentClient": {},
```

Comment:
@JBrenesSimpat Is this not needed? In the volume test runs, none of the student related endpoints are being tested (based on the TC build results). I haven't debugged locally, can you take a look?